docs/orchestration/concepts/artifacts.md

```diff
@@ -0,0 +1,96 @@
+
+## UI
+
+In the Prefect UI, to view artifacts created by a flow run, navigate to a specific flow run, then click the **Artifacts** tab.
+
+![Screenshot showing the Artifacts tab for a flow run](/orchestration/concepts/artifacts_flowrun.png)
+
+You can navigate between the artifacts created for a flow run by clicking the dots or arrows above the artifacts.
+
+Each artifact is identified by task that created it.
+
+To view the artifacts created by a task run, navigate to the specific task run, then click the **Artifacts** tab.
+
+![Screenshot showing the Artifacts tab for a task run](/orchestration/concepts/artifacts_taskrun.png)
+
+You can navigate between the artifacts created for a task run by clicking the dots or arrows above the artifacts.
+
+## Creating Link Artifacts
+
+To create link artifacts, just import `create_link_artifact` from `prefect.backend.artifacts`, then add a call to `create_link_artifact()` to any task orchestrated through a Prefect API. 
+
+::: tip Import path
+If you are using a version prior to Prefect 0.15.8, import `create_link` from `prefect.artifacts`.
+:::
+
+Pass `create_link_artifact()` a string containing the absolute URL of the location to which you want to link. 
+
+```python
+from prefect import task, Flow
+from prefect.backend.artifacts import create_link_artifact
+
+@task
+def send_artifact():
+    create_link_artifact("https://www.prefect.io/")
+```
+
+After the task runs, navigate to the UI’s Artifacts tab to see the output.
+
+## Creating Markdown Artifacts
+
+To create link artifacts, just import `create_markdown_artifact` from `prefect.backend.artifacts`, then add a call to `create_markdown_artifact()` to any task orchestrated through a Prefect API. 
+
+::: tip Import path
+If you are using a version prior to Prefect 0.15.8, import `create_markdown` from `prefect.artifacts`.
+:::
+
+Pass `create_markdown_artifact()` a string that will be rendered as an artifact. The string can contain any [Github-flavored Markdown](https://github.github.com/gfm/) markup including images, links, and tables. 
+
+Note that any images referenced in your markdown must be linked by the absolute URL of a publicly available image. Linking to local files or by relative URL is not supported.
+
+```python
+from prefect import task, Flow
+from prefect.backend.artifacts import create_markdown_artifact
+
+@task
+def send_artifact():
+    create_markdown_artifact("# Heading\nText with [link]("https://www.prefect.io/").")
+```
+
+`create_markdown_artifact()` accepts any valid Python string and variable interpolation including, for example, f-strings.
+
+```python
+@task
+def send_artifact(heading, link_text, link):
+    create_markdown_artifact(f"# {heading}\nLink to [{link_text}]({link}).")
+```
```

Comment:
This is a little confusing. It accepts a string. The formatting here happens before it is sent to the function. I'm worried this statement will lead to confusion where people think `create_markdown_artifact("{foo} {bar}", foo=1, bar=2)` will work.

Reply:
I can change this to a simpler example.